### ISSUE TITLE
KubeOne fails to install kubeadm on controlplane nodes with version 1.18.9

### ISSUE THREAD

**Issue by stroebitzer** [kubermatic/kubeone#1113]
**What happened**:

On using kubeone with the latest 1.18 kubeadm version (1.18.9) on vsphere I got the following error message:
```
+ sudo DEBIAN_FRONTEND=noninteractive apt-get install --option Dpkg::Options::=--force-confold --no-install-recommends -y kubelet=1.18.9-00 kubeadm=1.18.9-00 kubectl=1.18.9-00 kubernetes-cni=0.8.6-00 docker-ce=5:19.03.12~3-0~ubuntu-bionic docker-ce-cli=5:19.03.12~3-0~ubuntu-bionic
E: Unable to correct problems, you have held broken packages.
: Process exited with status 100  node=10.2.4.27
```

On executing the failing command within the VM I get the following:

```
ubuntu@run:~$ sudo DEBIAN_FRONTEND=noninteractive apt-get install --option Dpkg::Options::=--force-confold --no-install-recommends -y kubelet=1.18.9-00 kubeadm=1.18.9-00 kubectl=1.18.9-00 kubernetes-cni=0.8.6-00 docker-ce=5:19.03.12~3-0~ubuntu-bionic docker-ce-cli=5:19.03.12~3-0~ubuntu-bionic
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:
The following packages have unmet dependencies:
 kubeadm : Depends: kubernetes-cni (>= 0.8.7)
 kubelet : Depends: kubernetes-cni (>= 0.8.7)
E: Unable to correct problems, you have held broken packages.
```

**What is the expected behavior**:

Kubeone can install the packages on the master nodes and sets up K8s

**How to reproduce the issue**:

Create a cluster with version 1.18.9

**Anything else we need to know?**

**Information about the environment**:
KubeOne version (`kubeone version`): 1.0.0
Operating system: Linux Manjaro
Provider you're deploying cluster on: VSphere
Operating system you're deploying on: Ubuntu 20.04

**Comment by kron4eg** [comment]
@stroebitzer Please try the latest master 

@xmudrii seems like we need to backport https://github.com/kubermatic/kubeone/pull/1100

**Comment by xmudrii** [comment]
I've cherry-picked #1100 to the release branch, which should fix this issue. We are planning to cut another v1.0 release over the week.

/close
Fixed by #1116.

**Comment by stroebitzer** [comment]
That worked out:

```
[10.2.46.111] Preparing to unpack .../06-kubernetes-cni_0.8.7-00_amd64.deb ...
[10.2.46.111] Unpacking kubernetes-cni (0.8.7-00) ...
```
